routes/teams.js

```diff
@@ -32,27 +32,44 @@ router.get('/quiz', function(req, res) {
   var rb_team_id = req.query.rb_team_id;
   var league = req.query.league;
   var list_id = req.query.list_id;
+  var type = req.query.type;
 
   if (req.query.trending){
-    db.collection('teams').findOne( { _id : rb_team_id}, function (err, item){
-      if (item != null){
-        var league = item.league;
-        var api_team_id = item.team_id;
-        var quiz_name = item.market + ' ' + item.name;
-        createQuiz(rb_team_id, list_id, league, quiz_name, res, returnItem, api_team_id);
-      }
-      else{
-        res.status(500);
-        res.json('error', { error: 'unable to find a matching team' });
-      }
-    });
+    if (type != undefined){
```

Comment:
Make this type == leaders    so that we don't have to come back and change it in two days when we add a type somewhere else?